packages/headless-components/blog/src/react/core/Comments.tsx

```diff
@@ -0,0 +1,215 @@
+import { comments } from '@wix/comments';
+import { Sort as SortPrimitive } from '@wix/headless-components/react';
+import { useService } from '@wix/services-manager-react';
+import React from 'react';
+import type {
+  BlogPostCommentsServiceAPI,
+  BlogPostCommentsServiceConfig,
+  CommentWithResolvedFields,
+  QueryCommentsSort,
+} from '../../services/blog-post-comments-service.js';
+import { BlogPostCommentsServiceDefinition } from '../../services/blog-post-comments-service.js';
+
+export interface RootProps {
+  children: React.ReactNode;
+  commentsConfig?: BlogPostCommentsServiceConfig;
+}
+
+/**
+ * Core Blog.Post.Comments.Root component that provides Blog.Post.Comments.Root service context.
+ * This is the service-connected component that should be wrapped by the public API.
+ *
+ * @component
+ */
+export const Root = React.forwardRef<HTMLDivElement, RootProps>((props, ref) => {
+  const { children } = props;
+
+  return <div ref={ref}>{children}</div>;
+});
+
+Root.displayName = 'Blog.Post.Comments.Root (core)';
```

Comment:
As mentioned before, why do you need it to be Blog.Post.Comment and not just Comment?
remember that the user imports it from `@wix/blogs` so you don't need to have such prefix all the time.